spec/lib/auxiliary/test_helper_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require_relative '../../../lib/auxiliary/time_helper'
+
+RSpec.describe AutoHCK::Helper do
+  include AutoHCK::Helper
+
+  describe '#time_to_seconds' do
+    time_strings = {
```

Comment:
It is a bit awkward to name `time_strings` as it doesn't only have strings but also numbers. They are test cases so we can call `cases` or something like that.